pkg/spdk/replica.go

```diff
@@ -2689,3 +2691,25 @@ func (r *Replica) SetErrorState() {
 		needUpdate = true
 	}
 }
+
+// CleanupLvolTree retrieves the lvol tree with BFS. Then try its best effort to do cleanup bottom up.
+func (r *Replica) CleanupLvolTree(spdkClient *spdkclient.Client, rootLvolName string, bdevLvolMap map[string]*spdktypes.BdevInfo) {
```

Comment:
NIT: Can we keep this a util function and input the safe logger instead?

Reply:
Updated. Thank you.